docker-process.el

```diff
@@ -53,6 +57,11 @@
       (message command)
       (shell-command-to-string command))))
 
+(defun docker-use-format (go-template)
```

Comment:
This function does not make a lot of sense... why not simply this:

``` elisp
(data (docker-run "container ls" docker-container-ls-arguments (format "--format=\"%s\"" (if (docker--is-podman-p) fmt "json")))
```

Reply:
I would put this in the docker-utils file? I think this is a better place now for this. Also, I would rename this than to docker-utils-podman-p. What do you think?

Reply:
Yes, good idea.